scrapers/covid-19/govScrapers/getMexico.js

```diff
@@ -0,0 +1,110 @@
+const axios = require('axios');
+const logger = require('../../../utils/logger');
+const querystring = require('querystring');
+
+// a map of Mexican states with the corresponding IDs gob.mx uses server-side
+const stateIDs = [
+	{ state: 'aguascalientes', id: '01' },
+	{ state: 'baja california', id: '02' },
+	{ state: 'baja california sur', id: '03' },
+	{ state: 'campeche', id: '04' },
+	{ state: 'coahuila', id: '05' },
+	{ state: 'colima', id: '06' },
+	{ state: 'chiapas', id: '07' },
+	{ state: 'chihuahua', id: '08' },
+	{ state: 'ciudad de mexico', id: '09' },
+	{ state: 'durango', id: '10' },
+	{ state: 'guanajuato', id: '11' },
+	{ state: 'guerrero', id: '12' },
+	{ state: 'hidalgo', id: '13' },
+	{ state: 'jalisco', id: '14' },
+	{ state: 'estado de mexico', id: '15' },
+	{ state: 'michoacan', id: '16' },
+	{ state: 'morelos', id: '17' },
+	{ state: 'nayarit', id: '18' },
+	{ state: 'nuevo leon', id: '19' },
+	{ state: 'oaxaca', id: '20' },
+	{ state: 'puebla', id: '21' },
+	{ state: 'queretaro', id: '22' },
+	{ state: 'quintana roo', id: '23' },
+	{ state: 'san luis potosi', id: '24' },
+	{ state: 'sinaloa', id: '25' },
+	{ state: 'sonora', id: '26' },
+	{ state: 'tabasco', id: '27' },
+	{ state: 'tamaulipas', id: '28' },
+	{ state: 'tlaxcala', id: '29' },
+	{ state: 'veracruz', id: '30' },
+	{ state: 'yucatan', id: '31' },
+	{ state: 'zacatecas', id: '32' }
+];
```

Comment:
can't we simplify this to:

```js
const states = [
	'aguascalientes',
	'baja california',
	'baja california sur',
	'campeche',
	'coahuila',
	'colima',
	'chiapas',
	'chihuahua',
	'ciudad de mexico',
	'durango',
	'guanajuato',
	'guerrero',
	'hidalgo',
	'jalisco',
	'estado de mexico',
	'michoacan',
	'morelos',
	'nayarit',
	'nuevo leon',
	'oaxaca',
	'puebla',
	'queretaro',
	'quintana roo',
	'san luis potosi',
	'sinaloa',
	'sonora',
	'tabasco',
	'tamaulipas',
	'tlaxcala',
	'veracruz',
	'yucatan',
	'zacatecas'
];
```

Reply:
This would cause the scraper to rely on this `states` array being ordered alphabetically. That's a kind of unintuitive restriction that may not be known to someone looking at this later on.

Having the `id` property hardcoded there on the other hand makes it clear that there's a strict association between each `state` and its `id`, as far as the source is concerned, rather than an arbitrary index.